pxr/usd/usdLux/lightAPI.h

```diff
@@ -46,6 +46,41 @@ class SdfAssetPath;
 /// type (e.g. RectLight or DistantLight) or is applied directly to a Gprim 
 /// that should be treated as a light.
 /// 
+/// <b>Quantities and Units</b>
+/// 
+/// Most renderers consuming OpenUSD today are RGB renderers, rather than
+/// spectral. Units in RGB renderers are tricky to define as each of the red,
+/// green and blue channels transported by the renderer represents the
+/// convolution of a spectral exposure distribution, e.g. CIE Illuminant D65,
+/// with a sensor response function, e.g. CIE 1931 𝓍̅. Thus the main quantity
+/// in an RGB renderer is neither radiance nor luminance, but "integrated
+/// radiance" or "tristimulus weight".
+/// 
+/// The emission of a default light with `intensity` 1 and `color` [1, 1, 1] is
+/// an Illuminant D spectral distribution with chromaticity matching the
+/// rendering color space white point, normalized such that a ray normally
+/// incident upon the sensor with EV0 exposure settings will generate a pixel
+/// value of [1, 1, 1] in the rendering color space.
+/// 
```

Comment:
Is this possible with radiometric units? I'd prefer that we move away from standard observers and photometry where possible, and more directly to the realm of radiometry so that we can just deal with measurements of energy, not measurements mapped to human vision.

Reply:
It's much easier to talk about expected values in an RGB renderer using photometric units, which is of course 99% of renderers out there.

Reply:
Apologies, I was being bone headed here. You specifically said the luminaire has an Illuminant D spectral distribution with chromaticity matching the rendering color space white point. The observer is the sensor response function. I have no idea what I was on about, this is already the way I was hoping it would be.